mod/cornercutter/ModFeature/GlobalSetting/CoWorkingSpaceSlide.cs

```diff
@@ -0,0 +1,118 @@
+﻿using cornercutter.Enum;
+using cornercutter.ModLoader;
+using HarmonyLib;
+using System;
+using System.Collections;
+using UnityEngine;
+
+namespace cornercutter.ModFeature.GlobalSetting
+{
+    // In order to send Jackie to CWS from the slide, we are intercepting the trigger
+    // that is fired to have her exit it.
+    [HarmonyPatch(typeof(Animator), nameof(Animator.SetTrigger), new Type[] { typeof(string) })]
+    class CoWorkingSpaceSlide
+    {
+        // Flag for checking if we have been inside the slide, and
+        // so whether to run a special animation next time we go back to Fizzle
+        public static bool runAssociatedSlideAnimationNext = false;
```

Comment:
This is repeated below, but I'm not a fan of just having public, non-constant fields. I think this should probably be a property:
```cs
public static bool RunAssociatedSlideAnimationNext { get; set; };
```